src/utils/orgUtils.ts

```diff
@@ -65,6 +72,103 @@ export class OrgUtils {
         }
     }
 
+    public static async getAllCompactLayoutsForSObject(
+        sObjectName: string
+    ): Promise<any> {
+        const org = await Org.create();
+        const conn = org.getConnection();
+
+        const result = await conn.request(
+            `/services/data/v59.0/sobjects/${sObjectName}/describe/compactLayouts`
+        );
+
+        return Promise.resolve(result);
+    }
+
+    public static async getCompactLayoutForSObject(
+        sObjectName: string,
+        recordTypeId: string
+    ): Promise<any> {
+        const org = await Org.create();
+        const conn = org.getConnection();
+
+        const result = await conn.request(
+            `/services/data/v59.0/sobjects/${sObjectName}/describe/compactLayouts/${recordTypeId}`
+        );
+
+        return Promise.resolve(result);
+    }
+
+    public static async getCompactLayoutFieldsForSObject(
+        sObjectName: string
+    ): Promise<CompactLayoutField[]> {
+        try {
+            const fields: CompactLayoutField[] = [];
+
+            // Get all the compact layouts associated to this sObject first
+            let result = await this.getAllCompactLayoutsForSObject(sObjectName);
+
+            if (result) {
+                const resultObj = result as Object;
```

Comment:
If we're assuming any value for `result` is an `Object`, can `getAllCompactLayoutsForSObject()` return `Promise<Object>` instead of `Promise<any>`?